app/views/cities/show.html.haml

```diff
@@ -1,7 +1,12 @@
-%h1= "#{@city.name} Tours"
+.container
+  %h1.center-align= "#{@city.name} Tour"
 
-
-%div
-  - @city.categories.each do |category|
-    = link_to "Entertainment", city_category_path(@city.id, category.id)
-    = link_to "Food & Drink", city_category_path(@city.id, category.id)
+  %h4.center-align Categories
+  %br
+  .container
+    - @city.categories.each do |category|
+      .row
+        %div{class: "col s8 offset-s2"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names